frappe/core/doctype/user/user.py

```diff
@@ -788,7 +788,7 @@ def sign_up(email, full_name, redirect_to):
 			return 2, _("Please ask your administrator to verify your sign-up")
 
 @frappe.whitelist(allow_guest=True)
-@rate_limit(key='user', limit=get_password_reset_limit, seconds = 24*60*60, methods=['POST'])
+@rate_limit(limit=get_password_reset_limit, seconds = 24*60*60, methods=['POST'])
```

Comment:
Why is this removed?

Reply:
Because the key was a mandatory parameter and one of the researchers demonstrated that the rate-limit on reset password or any other method can be bypassed with different keys each time. The researcher did this by sending around 50+ requests to reset passwords with different keys each time and all gave HTTP 200 responses.